doc/cookbook/exclusion_strategies.rst

```diff
@@ -57,20 +57,21 @@ expose them via an API that is consumed by a third-party:
     class VersionedObject
     {
         /**
-         * @Until("1.0.x")
```

Comment:
I would not abandon `@Until` and `@Since`, mainly for performance reasons. my guess is that `composer/semver` is way slower that the php's `version_compare` function, in a large object graph in it can make the difference.

BTW, i would be curious to see a benchmark about it